app/operator/konnect/reference/faq.md

```diff
@@ -48,12 +48,12 @@ The reconciliation loop runs once per minute. Wait 60 seconds, then refresh the
 
 ## Can I use Secrets for Consumer credentials like in {{ site.kic_product_name }}?
 
-{{ site.gateway_operator_product_name }} uses new `Credential` CRDs for managing [consumer credentials](/operator/konnect/crd/gateway/consumer/). We plan to support Kubernetes `Secret` resources in a future release. [#618](https://github.com/Kong/gateway-operator/issues/618).
+{{ site.operator_product_name }} uses new `Credential` CRDs for managing [consumer credentials](/operator/konnect/crd/gateway/consumer/). We plan to support Kubernetes `Secret` resources in a future release. [#618](https://github.com/Kong/gateway-operator/issues/618).
```

Comment:
Same, this has already been done in https://github.com/Kong/kong-operator/issues/618.

Old docs PR: https://github.com/Kong/docs.konghq.com/issues/8474.